src/backend/joanie/core/models/products.py

```diff
@@ -475,6 +475,12 @@ class Order(BaseModel):
         help_text=_("User has consented to the platform terms and conditions."),
         db_column="has_consent_to_terms",
     )
+    has_waived_withdrawal_right = models.BooleanField(
+        verbose_name=_("has waived to withdrawal right"),
+        editable=False,
+        default=False,
+        help_text=_("User has waived to its withdrawal right."),
```

Comment:
nit:
```suggestion
        help_text=_("User has waived his right of withdrawal."),
```

Reply:
Just use `their` instead of `his` to be gender neutral.